docs/architecture.md

```diff
@@ -6,27 +6,31 @@ This is how Style Dictionary works under the hood.
 
 Let's take a closer look into each of these steps.
 
-## 1. Find all token files
+## 1. Read the config
```

Comment:
Maybe better "Parse the config"? otherwise looks like **I** have to read the config :)